### PR TITLE
Add ansible.md

### PR DESCRIPTION
**Pull request by mperezv**
Second version of ansible.md

### CODE REVIEW

**Review by sauloperez — APPROVED** [review]

**Comment by sauloperez** [comment]
What happens then with https://github.com/coopdevs/handbook/pull/4 ? Does this one contain those changes?

**Comment by mperezv** [comment]
@sauloperez It should, if I did everything right.

File: devenv.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# devenv
```
Comment (danypr92) [inline devenv.md]: What do you think about to move this file to another MR? One for the devenv documentation.

File: ansible.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## ¿Por qué es útil?
+Porque nos permite montar máquinas con software concreto y configuraciones concretas de forma automática y sencilla. Ahorrándonos así tiempo y complejidad en las instalaciones.
+
+## ¿Cómo funciona?
+Para Ansible existen dos tipos de máquinas, el *control node* y el *managed node* o *host*. 
+El *control node* es la máquina desde la que comienza la orquestación, desde la que lanzamos las órdenes de Ansible y donde éste está instalado. El *managed node* es la máquina orquestada, es decir, aquella a la que accedemos por SSH y dónde se ejecutan las órdenes de Ansible, debe tener instalado Python.
+
+### Modules
+Ansible está organizado en módulos que son las unidades de código que ejecuta. Se puede invocar un solo módulo con `ansible -m <module-name> o varios diferentes en un *playbook*. Existen módulos predefinidos que Ansible facilita, pero también se pueden crear propios.
```
Comment (danypr92) [inline ansible.md]:
```suggestion
Ansible está organizado en módulos que son las unidades de código que ejecuta. Se puede invocar un solo módulo con `ansible -m <module-name>` o varios diferentes en un *playbook*. Existen módulos predefinidos que Ansible facilita, pero también se pueden crear propios.
```

File: ansible.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### Playbook
+El *playbook* es la lista de *tasks* escrita en [YAML](https://es.wikipedia.org/wiki/YAML), ordenada y guardada para que se puedan ejecutar en un de forma secuencial repetidamente. Puede contener también variables.
+
+### Roles
+Los roles son grupos de ficheros -como *templates* y variables- y *tasks* basados en una estructura de ficheros concreta. Existen roles de Ansible y roles que aporta la comunidad en [galaxy](https://galaxy.ansible.com/).
+
+### Tasks
+Son las unidades de acción en Ansible.
+Se definen en Tasks/main.yml dentro de la carpeta de cada rol.
```
Comment (danypr92) [inline ansible.md]:
```suggestion
Se definen en `tasks/main.yml` dentro de la carpeta de cada rol.
```

**Review by danypr92 — CHANGES_REQUESTED** [review]

**Comment by mperezv** [comment]
Finally, we decided to add it to our wiki [here](https://github.com/coopdevs/handbook/wiki/Uso-de-Ansible-en-Coopdevs).

**Pull request closed without merging**